wire/msgcfiltersv2.go

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2019-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+)
+
+// MaxCFiltersV2PerBatch is the maximum number of committed filters that may
+// be sent on a cfiltersv2 message.
+//
+// This number has been decided assuming a sequence of blocks filled with
+// transactions specifically designed to maximize the filter size, such that
+// a cfiltersv2 message will not be larger than the maximum allowed P2P message
+// size on any of the currently deployed networks.
+const MaxCFiltersV2PerBatch = 100
```

Comment:
Incidentally, at least on my informal tests, going larger than this doesn't really bring any detectable benefits.